src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
It makes no sense to extends a class with a static method and call the method through the class (line 10).
It is better to make the method non-static and create an instance of the class to call this method